src/index.js

```diff
@@ -101,8 +101,10 @@ function pingServer (address) {
 /**
  * Reads the arp table.
  */
-function arpAll () {
-  return cp.exec('arp -a', options).then(parseAll)
+function arpAll (_, skipNameResolution = false) {
```

Comment:
As the first param (`servers`) is not used, we can safely remove it here and in line 41 as well.

Reply:
Removed this param. (Not sure why I added it in the first place)